plugin.js

```diff
@@ -72,6 +72,7 @@ module.exports = plugin(
           'box-shadow': `var(--tw-ring-offset-shadow), var(--tw-ring-shadow), var(--tw-shadow)`,
           'border-color': theme('colors.blue.600', colors.blue[600]),
         },
+        '@apply bg-gray-50 border border-gray-300 text-gray-900 rounded-md focus:ring-blue-500 focus:border-blue-500 w-full dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500': {},
```

Comment:
I would prefer this stays where it is as this section where you moved the styles too came from Flowbite's default but I intend to remove/consolidate those before final release. I'd prefer the original location be updated with the missing selectors to match.

```suggestion
```